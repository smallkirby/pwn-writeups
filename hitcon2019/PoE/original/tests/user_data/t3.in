i 0 abcde
r 1 2 A
R 0 4
d 0 5
n
i 0 123456
D 2 3
d 0 3
R 0 3
c 0 1
s 0
p 0
d 0 6
n
i 0 abcdefg
R 3 4
d 0 7
r 0 7 A
d 4 3
i 0 q
d 0 3
