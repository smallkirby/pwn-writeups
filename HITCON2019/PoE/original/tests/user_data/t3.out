Done.
Done.
Done.
dAAae
Switched to new tab 1.
Done.
Done.
126
Done.
Done.
Switched to tab 0.
Done.
6dAAae
Switched to new tab 2.
Done.
Done.
abcgfed
Done.
AAA
Done.
qAA
